app/validators/profile_validator.rb

```diff
@@ -32,6 +32,9 @@ def validate(record)
   private
 
   def summary_too_long?(record)
+    # During onboarding, the attribute is not yet available
```

Comment:
I would remove this comment, it's misleading. If the attribute isn't available during onboarding, it won't be available after either, no fields get created in the process.